opentelemetry-exporter-geneva/geneva-uploader/Cargo.toml

```diff
@@ -15,7 +15,7 @@ native-tls = "0.2"
 thiserror = "2.0"
 chrono = "0.4"
 url = "2.2"
-
+lz4_flex = { version = "0.11", features = ["safe-encode"], default-features = false }
```

Comment:
Should this be behind a feature or should it always be included?

Reply:
It needs to be always included, as it is required for the actual compression.

Reply:
Yes, if you use the compression it needs to be included, but right now the compression is dead code. So my question is if the compression will always be used so you always need to include this.

Reply:
Understood - compression will definitely be used as part of #261.